pg_replicate/src/pipeline/sources/postgres.rs

```diff
@@ -69,6 +69,7 @@ impl PostgresSource {
         let (table_names, publication) =
             Self::get_table_names_and_publication(&replication_client, table_names_from).await?;
         let table_schemas = replication_client.get_table_schemas(&table_names).await?;
+        replication_client.commit_txn().await?;
```

Comment:
This is the fix for #80, but making sure we don't have a dangling transaction going out of the constructor. I'm not sure whether we need the transaction at all here, but erred on the side of caution.